spec/models/user_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  subject { User.new(name: 'John', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Random bio', postsCounter: 2) }
+
+  before { subject.save }
+
+  it 'should save the data' do
+    expect(subject).to be_valid
+  end
+
+  it 'name should be present' do
+    subject.name = nil
+    expect(subject).to_not be_valid
+  end
+end
```

Comment:
- [ ] Kindly add tests for other [validations](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=PostsCounter%20must%20be%20an%20integer%20greater%20than%20or%20equal%20to%20zero.) and for  methods.

[Requirement](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/blog_app_validations_model_spec.md#:~:text=Add%20unit%20specs%20for%20all%20of%20your%20models%27%20methods%20and%20validations.)